src/components/layout/header/AdaptiveMenu.jsx

```diff
@@ -99,7 +137,14 @@ const AdaptiveMenu = ({ toggleMobileMenu, mobileMenu }) => {
 
       return true;
     });
-  }, [executeLoggingInProcess, handleAddQuestion, logout, token]);
+  }, [
+    executeLoggingInProcess,
+    handleAddQuestion,
+    logout,
+    profile.avatar?.thumbnail,
+    profile.name,
```

Comment:
давай просто `profile` передадим, будет лаконичнее

```suggestion
    profile
```